File: Github-Users/multi/SecureSetHQ/crackme1.c
#include <stdlib.h>
#include <stdio.h>

int check_pw(char * guess)
{
  // Password is letmein
  char pw[] = "letmein";
  // Check if the password matches the pw string
  return (0 == strcmp(pw, guess));
}

int main()
{
  char buf[64] = {0};
  printf("PASSWORD: ");
  scanf("%63s", buf);

  if (check_pw(buf))
	  printf("ACCESS GRANTED!\n");
  else
	  printf("ACCESS DENIED!\n");
	  
  return 0;
}

File: Github-Users/multi/SecureSetHQ/binaries/crackme1.py
#!/usr/bin/python

def check_pw(guess):
    '''Tests the user's guess against the secret password'''
    p1 = 'pan'
    p2 = 'cakes'
    return guess == p1 + p2

guess = raw_input("PASSWORD: ")
if check_pw(guess):
    print "ACCESS GRANTED"
else:
    print "DENIED!"
